jicofo/src/main/kotlin/org/jitsi/jicofo/conference/Participant.kt

```diff
@@ -402,8 +404,18 @@ open class Participant @JvmOverloads constructor(
         }
         override fun onSessionTerminate(jingleSession: JingleSession, iq: JingleIQ) =
             conference.onSessionTerminate(jingleSession, iq)
-        override fun onTransportInfo(jingleSession: JingleSession, contents: List<ContentPacketExtension>) =
-            conference.onTransportInfo(jingleSession, contents)
+        override fun onTransportInfo(
+            jingleSession: JingleSession,
+            contents: List<ContentPacketExtension>
+        ): StanzaError? {
+            checkJingleSession(jingleSession)?.let { return it }
+
+            val transport = contents.getTransport()
+                ?: return StanzaError.from(StanzaError.Condition.bad_request, "missing transport active").build()
```

Comment:
I don't understand "active" in this message.  Typo?  Is this logged?

Reply:
It's not logged, but it will be returned to the client. Do you think I should log it? Just this one or any Jingle error we return?

Reply:
Maybe every Jingle error?  It's worth thinking about at least.

Reply:
I added a commit, it might be quite verbose to print both the request and response xml, but I don't expect these to happen often